server/pwa/manifest.js

```diff
@@ -34,7 +34,6 @@ const buildManifest = ( { branchName } ) => {
 		display: 'standalone',
 		gcm_sender_id: '87234302238',
 		background_color: '#0078be',
-		theme_color: '#0078be',
```

Comment:
what does this control? does the theme-color in the meta supercede it? Did we ever actually need it?

Reply:
This is another way to do the same thing on a static site-wide basis.  If we want to set anything dynamically (in JS), we'll have to use the `meta` header method.

See:
* https://developers.google.com/web/fundamentals/web-app-manifest/#theme-color
* https://developers.google.com/web/updates/2015/08/using-manifest-to-set-sitewide-theme-color

Reply:
Also, this value is transposed of another in the codebase: (`#0078be` vs `#0087be`).  Not sure if intentional.